src/validators/endpoints.py

```diff
@@ -56,27 +55,26 @@ async def register_validators(
 @router.post('/fund')
 async def fund_validators(
     request: schema.ValidatorsFundRequest,
-) -> schema.ValidatorsFundResponse:
-    validator_items = []
-    app_state = AppState()
-    if not app_state.keystore:
-        raise ValueError('Keystore is required for funding validators')
-
-    validators = get_validators_for_funding(
-        keystore=app_state.keystore,
-        vault_address=request.vault,
-        public_keys=request.public_keys,
-        amounts=request.amounts,
-    )
-
-    for validator in validators:
-        validator_items.append(
-            schema.ValidatorsFundResponseItem(
-                public_key=validator.public_key,
-                deposit_signature=validator.deposit_signature,
-                amount=validator.amount,
-            )
+) -> schema.ValidatorsSignatureResponse:
+    validators = []
+
+    # use empty signature and withdrawal credentials for funding
+    empty_signature = bytes(96)
+    empty_withdrawal_credentials = bytes(32)
+    for public_key, amount in zip(request.public_keys, request.amounts):
+        deposit_data = DepositData(
+            pubkey=Web3.to_bytes(hexstr=public_key),
+            withdrawal_credentials=empty_withdrawal_credentials,
```

Comment:
Lets use vault address here, otherwise the transaction will be rejected due to deposit_data_root mismatch.